cli/azd/resources/apphost/templates/resources.bicept

```diff
@@ -162,20 +162,36 @@ resource {{bicepName $name}} 'Microsoft.App/containerApps@2023-05-02-preview' =
         allowInsecure: {{$value.Ingress.AllowInsecure}}
 {{- end}}
       }
+{{- end}}
+{{- if gt (len $value.Secrets) 0 }}
+      secrets: [
+{{- range $sName, $sValue := $value.Secrets}}
+        {
+          name: '{{containerAppSecretName $sName}}'
+          value: {{toDotNotation $sValue}}
+        }
+{{- end}}    
+      ]
 {{- end}}
     }
     template: {
       containers: [
         {
           image: '{{$value.Image}}'
-          name: '{{$name}}'
-{{- if $value.Env }}
+          name: '{{containerAppName $name}}-${resourceToken}'
```

Comment:
Did we intend to add the ${resourceToken} here?  I'm not sure we need it to uniquify the container name itself inside the template?

Reply:
will remove. This was a mistake